src/utils/components/ApplyStorageProfileSettingsCheckbox/ApplyStorageProfileSettingsCheckbox.tsx

```diff
@@ -1,37 +1,26 @@
-import * as React from 'react';
+import React, { FC } from 'react';
 
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { Checkbox, FormGroup } from '@patternfly/react-core';
 
-import { diskReducerActions, DiskReducerActionType } from '../state/actions';
-import { DiskFormState } from '../state/initialState';
-
 type ApplyStorageProfileSettingsCheckboxProps = {
   claimPropertySets: {
```

Comment:
u can change to ClaimPropertySets type? as utils files below

Reply:
I created the type and didn't change it. 🤦🏼‍♂️ Fixed!